### PR TITLE
create sravya71.md

### PR DESCRIPTION
**Pull request by sravya71**
https://raw.githubusercontent.com/sravya71/Sravya.treehouses.github.io/Branch-sravya-new-branch/Sravya71.md

### CODE REVIEW

**Review by pattanawadee88 — CHANGES_REQUESTED** [review]
you have to put your profile in /pages/vi/profiles

**Comment by sravya71** [comment]
@pattanawadee88  The requested changes have been previewed and changed. Please check 
https://github.com/sravya71/Sravya.treehouses.github.io/blob/Branch-sravya-new-branch/pages/vi/profiles/Sravya71.md

Can I create a new pull request??

Thank you,
Sravya

**Review by sravya71 — COMMENTED** [review]
Sravya71.md Have been kept in /pages/vi/profiles

File: pages/vi/profiles/Sravya71.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+####_**Hi, I am Laxmi Sravya Nittala, I am an aspiring Intern for OLE Below are my Details**_:
+1.Name: Laxmi Sravya Nittala
+2.Location\Time Zone: NewJersey\EST
+3.OS Version: Windows 10 Home
+ 
+ >Hi **OLE Team**, I am Sravya, I have done my masters from ![Lamar University][Lamar], which is a state university in Texas.
+ >
+ >I have AWS, CCNA certifications.I am here to learn, Contribute and collaborate with the **OLE Team**. Looking forward to work **OLE**!
+ 
+ [Lamar]:https://www.google.com/search?q=lamar+university&sxsrf=ACYBGNQ54CQIHjUDPGh_Q5SXSsnXNVVNYA:1574975118782&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjTrfDD543mAhXSZd8KHTUGBHEQ_AUoA3oECBEQBQ#imgrc=zYiPE_4VWEHlzM:
```
Comment (pattanawadee88) [inline pages/vi/profiles/Sravya71.md]: a picture is not working

**Review by pattanawadee88 — CHANGES_REQUESTED** [review]
https://raw.githack.com/sravya71/Sravya.treehouses.github.io/Branch-sravya-new-branch/#!pages/vi/profiles/Sravya71.md

this is the right raw.githack link

File: pages/vi/profiles/Sravya71.md (hunk 1 above)
Comment (sravya71) [reply pages/vi/profiles/Sravya71.md]: Hi @pattanawadee88 , Attached below is the new raw link:

https://raw.githubusercontent.com/sravya71/Sravya.treehouses.github.io/Branch-sravya-new-branch/pages/vi/profiles/Sravya71.md

Comment (pattanawadee88) [reply pages/vi/profiles/Sravya71.md]: the link should be included #! before the word 'pages'

Comment (sravya71) [reply pages/vi/profiles/Sravya71.md]: @pattanawadee88  Below is the requested Link: 
https://raw.githack.com/sravya71/sravya.treehouses.github.io/Branch-sravya-new-branch/#!pages/vi/profiles/Sravya71.md

Thank you,
Sravya.

Comment (pattanawadee88) [reply pages/vi/profiles/Sravya71.md]: the picture is still not working for me. 
you can check this https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images for reference

Comment (sravya71) [reply pages/vi/profiles/Sravya71.md]: @pattanawadee88 https://raw.githack.com/sravya71/Sravya.treehouses.github.io/Branch-sravya-new-branch/#!pages/vi/profiles/Sravya71.md  

Can you please check this link??

**Review by pattanawadee88 — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Comment by dogi** [comment]
https://raw.githack.com/sravya71/Sravya.treehouses.github.io/Branch-sravya-new-branch/#!pages/vi/profiles/sravya71.md

**Pull request merged**